content/components/checkbox-group.mdx

```diff
@@ -0,0 +1,25 @@
+---
+title: Checkbox group
+figma: https://www.figma.com/file/GCvY3Qv8czRgZgvl1dG6lp/Primer-Web?node-id=15341%3A47574
+react: https://primer.style/react/CheckboxGroup
+description: Checkbox group renders a set of checkboxes.
+---
+
+<img
+  width="960"
+  alt="An image of a checkbox group showing two selections"
+  src="https://user-images.githubusercontent.com/586552/205333350-f767c1ee-61b9-49e8-963d-0fad537749e9.png"
+/>
+
+## Usage
+
+Use checkbox group to allow users to select multiple items from a list of individual items, or to mark one individual item as selected.
+
+## Related links
+
+- [Forms](../ui-patterns/forms#related-components)
```

Comment:
I'm not sure if we need to specifically link to the `#related-components` anchor of that page